subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -117,6 +117,59 @@ PMD has been updated to https://pmd.github.io/pmd-6.48.0/pmd_release_notes.html[
 When configuring an executable explicitly for link:{groovyDslPath}/org.gradle.api.tasks.compile.ForkOptions.html#org.gradle.api.tasks.compile.ForkOptions:executable[`JavaCompile`] or link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html#org.gradle.api.tasks.testing.Test:executable[`Test`] tasks, Gradle will now emit an error if this executable does not exist.
 In the past, the task would be executed with the default toolchain or JVM running the build.
 
+==== Changes to dependency declarateions in Test Suites
```

Comment:
I still feel like this is too vague. I would prefer something closer to my suggested title that makes the "changes" explicit. Also we should fix the typo here.